pyvelox/CMakeLists.txt

```diff
@@ -15,12 +15,12 @@
 if(VELOX_BUILD_PYTHON_PACKAGE)
   message("Creating pyvelox module")
   include_directories(SYSTEM ${CMAKE_SOURCE_DIR})
-  add_definitions(-DCREATE_PYVELOX_MODULE)
+  add_definitions(-DCREATE_PYVELOX_MODULE -DVELOX_DISABLE_GOOGLETEST)
```

Comment:
I had some issue with `Memory.h` including `GTestMacros.h` and it complaining that it couldn't find `gtest_prod.h`. I figured we didn't use gtest in the python bindings, and it was easier to add this instead of adding gtest to the include path.